test/client/fixtures/runner/api/select-test/index-test.js

```diff
@@ -232,6 +232,11 @@ $(document).ready(function () {
     if (browserUtils.isIE9)
         $(window.top.document).find('body').css('marginTop', '0px');
 
+    // NOTE: https://github.com/DevExpress/testcafe/issues/471
```

Comment:
we can remove comment here because we comment it with this hack function definition